js/tests/components/layout/main-wrapper/index.spec.js

```diff
@@ -1,5 +1,12 @@
 import Component from '@/components/layout/main-wrapper/index'
 
+// Mock manually this component because it changes randomly and breaks the snapshot
+jest.mock('@/components/layout/main-wrapper/main-content', () => {
```

Comment:
Do you think we can mock the `greeting` computed property instead?

Reply:
I'm not sure how to do that. It's a dependency's dependency's computed prop. MainContent is even dynamically imported, which I guess is why `@vue/test-utils` is not stubbing it despite the `shallowMount()`.

It looks like `@vue/test-utils` stubs dynamic imported components in [v1.0.0-beta.27](https://github.com/vuejs/vue-test-utils/blob/dev/CHANGELOG.md#100-beta27-2018-12-09)
Upgrading it would let us remove this manual mock and have a snapshot that does not change all the time. What do you think about it ?

Reply:
Of course when I say "let's upgrade `@vue/test-utils`", I mean I'll try and submit another PR to keep this one small and easy to review.

Reply:
OK let's ignore this for now :)